src/assetManager.js

```diff
@@ -1,29 +1,24 @@
+import ImageManager from './assets/imageManager';
+
 export default class AssetManager {
   constructor() {
     this.assets = {
-      images: {}
+      images: new ImageManager(),
```

Comment:
I like this - as we'll later have multiple asset types, not just images 👍

Reply:
Definitely at least one for sound effects, and animations will need a custom image manager.